app/views/visualizerModal.html

```diff
@@ -0,0 +1,35 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="cancel()" aria-hidden="true">&times;</button>
+  <h4 class="modal-title">
+    <i class="glyphicon glyphicon-th"></i>&nbsp;&nbsp;
+    Visualizations Management
+
+    <!-- hasErrors Alerts -->
+    <alert style="display: inline-block; margin: 0px 0px 0px 5px; padding: 8px;" ng-repeat="alert in alerts.hasErrors" type="alert.type">
+      <i class="glyphicon glyphicon-warning-sign"></i>{{alert.msg}}
+    </alert>
+  </h4>
+</div>
+<div class="modal-body">
+  
+
+  <!-- include for visualizer settings view -->
+  <div ng-include="'views/partials/visualizer-settings.html'"></div>
+
+</div>
+
+<div class="modal-footer">
+  <div>
+    <!-- hasErrors Alerts -->
+    <alert style="display: inline-block; margin: 0px 0px 0px 5px; padding: 8px;" ng-repeat="alert in alerts.hasErrors" type="alert.type">
+      <i class="glyphicon glyphicon-warning-sign"></i>{{alert.msg}}
+    </alert>
+    <!-- hasErrors Alerts -->
+    <button type="button" class="btn btn-default" ng-click="cancel()">Close</button>
+    
+    <button type="button" class="btn btn-primary" ng-click="submitSettingsForm()">
+      <span ng-if="update == true">Save changes</span>
+      <span ng-if="update == false">Create Visualizer</span>
```

Comment:
I've noticed some issues when update isn't set yet and no text is displayed. Maybe change this just to `ng-if="update"` and `ng-if="!update"`, then I think it will work with null and undefined to.